frontend/public/components/service-account.jsx

```diff
@@ -110,10 +110,10 @@ const menuActions = [
 const kind = 'ServiceAccount';
 
 const tableColumnClasses = [
-  classNames('col-sm-4', 'col-xs-6'),
-  classNames('col-sm-4', 'col-xs-6'),
-  classNames('col-md-2', 'hidden-sm', 'hidden-xs'),
-  classNames('col-sm-2', 'hidden-xs'),
+  '',
+  '',
+  'pf-m-hidden pf-m-visible-on-lg pf-u-w-16-on-lg',
+  'pf-m-hidden pf-m-visible-on-md pf-u-w-16-on-md',
```

Comment:
These classnames doesn't map with the existing classnames

Reply:
Well, it does I believe. breakpoint `sm` in bootstrap is `md` in patternfly and`md` is `lg`
So, first two r default width columns and the second two have corresponding classes
![Screenshot from 2021-06-21 20-01-05](https://user-images.githubusercontent.com/24852534/122779398-72bb3e80-d2cb-11eb-8ae4-cdea00afa508.png)

Reply:
If you are in a namespaced  page, then `secret` and `created` column is pushed to its right (originally it was evenly spaced)
![image](https://user-images.githubusercontent.com/9964343/122787619-2247df00-d2d3-11eb-9cad-c3409fc6cc75.png)

 so if you are in 980px then the column names are getting truncated even though there is a room to show two column names 
 
![image](https://user-images.githubusercontent.com/9964343/122788650-2a544e80-d2d4-11eb-9640-04c8dfd24d39.png)

Reply:
Thanks for pointing it out...Well, I tried a bit and a direct conversion of the existing classes dont quite work well, so I fine tuned keeping in mind various viewport sizes

![Screenshot from 2021-06-22 15-25-04](https://user-images.githubusercontent.com/24852534/122905294-b9ae4000-d36e-11eb-9544-076e771c743f.png)
![Screenshot from 2021-06-22 15-25-18](https://user-images.githubusercontent.com/24852534/122905301-badf6d00-d36e-11eb-9c55-0c066e1f0a4d.png)
![Screenshot from 2021-06-22 15-25-37](https://user-images.githubusercontent.com/24852534/122905305-bb780380-d36e-11eb-914e-a3eb2ce28d8e.png)